app/views/test.scala.html

```diff
@@ -55,7 +55,31 @@
ity:visible}.contributions-landing-ticker{display:-webkit-flex;display:flex;-webkit-flex-direction:column;flex-direction:column}@@media (max-width:46.24em){.contributions-landing-ticker{display:none}}.contributions-landing-ticker__values{display:-webkit-flex;display:flex;-webkit-justify-content:space-between;justify-content:space-between;-webkit-align-items:flex-end;align-items:flex-end}.contributions-landing-ticker__goal,.contributions-landing-ticker__so-far{display:-webkit-flex;display:flex;-webkit-flex-direction:column;flex-direction:column}.contributions-landing-ticker__so-far .contributions-landing-ticker__count{font-family:GT Guardian Titlepiece,Georgia,serif;font-size:2rem;font-weight:800;color:#ff7f0f}.contributions-landing-ticker__goal{-webkit-align-items:flex-end;align-items:flex-end}.contributions-landing-ticker__goal .contributions-landing-ticker__count{font-weight:700}.contributions-landing-ticker__label{font-style:italic}.contributions-landing-ticker__progress-bar{position:relative}.contributions-landing-ticker__progress{width:100%;height:.625rem;overflow:hidden;background-color:#dcdcdc;position:absolute}.contributions-landing-ticker__filled-progress{background-color:#ff7f0f;position:absolute;top:0;left:0;right:0;bottom:0;-webkit-transform:translateX(-100%);transform:translateX(-100%);transition:-webkit-transform 3s cubic-bezier(.25,.55,.2,.85);transition:transform 3s cubic-bezier(.25,.55,.2,.85);transition:transform 3s cubic-bezier(.25,.55,.2,.85),-webkit-transform 3s cubic-bezier(.25,.55,.2,.85)}.button--right-arrow{width:100%;padding:0 3.125rem 0 1.25rem;text-align:left}.button--right-arrow .svg-arrow-right-straight{left:calc(100% - 2.125rem);height:1rem}.stripe-payment-request-button__button{margin-top:1.5rem;margin-bottom:1.25rem}.stripe-payment-request-button__divider{line-height:2rem;padding:0;position:relative;overflow:hidden;text-align:center}.stripe-payment-request-button__divider:after,.stripe-payment-request-button__divider:before{content:" ";position:absolute;top:50%;margin-left:-999em;height:.125rem;width:998em;border-top:.0625rem solid #dcdcdc}.stripe-payment-request-button__divider:after{left:auto;width:999em;margin:0 0 0 1em}.stripe-payment-request-button--hidden{display:none}.component-contributions-survey{border-bottom:.0625rem solid #dcdcdc;margin:0 -.625rem;padding:0 .625rem}.component-contributions-survey h3{font-family:Guardian Text Egyptian Web,Georgia,serif;letter-spacing:-.03125rem;font-weight:900;font-size:1.0625rem;margin:.25rem 0}.component-contributions-survey .component-contributions-survey__button{display:inline-block;margin-bottom:2.25rem;margin-top:.75rem}.component-contributions-survey .component-contributions-survey__button .button--survey{background-color:#121212;color:#fff}.component-contributions-survey .component-cta-link__text{padding-right:1.5625rem;font-size:1rem}body{min-height:100vh}.gu-content{position:relative;z-index:0;overflow:hidden}.gu-content--contribution-thankyou{background:#ffe5
```

Comment:
the actual changes are down here!  not minified to make it easier to port them later. but I have reduced the indentation to reduce the amount of redundant characters